From 41b3e7503247034ff52a592390942d072df4b5f9 Mon Sep 17 00:00:00 2001
From: Xaver Hugl <xaver.hugl@gmail.com>
Date: Mon, 11 Nov 2024 13:01:20 +0100
Subject: [PATCH] outputconfigurationstore: default to internal display scale
 1.0

This is needed until some issues with the default scale algorithm are sorted out
---
 src/outputconfigurationstore.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/outputconfigurationstore.cpp b/src/outputconfigurationstore.cpp
index 0e735d9d63..7b35d15285 100644
--- a/src/outputconfigurationstore.cpp
+++ b/src/outputconfigurationstore.cpp
@@ -531,6 +531,9 @@ std::shared_ptr<OutputMode> OutputConfigurationStore::chooseMode(Output *output)
 
 double OutputConfigurationStore::chooseScale(Output *output, OutputMode *mode) const
 {
+    if (output->isInternal()) {
+        return 1.0;
+    }
     if (output->physicalSize().height() < 3 || output->physicalSize().width() < 3) {
         // A screen less than 3mm wide or tall doesn't make any sense; these are
         // all caused by the screen mis-reporting its size.
-- 
2.47.0

